pkg/hash/reader.go

```diff
@@ -117,6 +118,11 @@ func (r *Reader) MD5HexString() string {
 	return hex.EncodeToString(r.md5sum)
 }
 
+// MD5Base64String return base64 encoded MD5sum value.
+func (r *Reader) MD5Base64String() string {
```

Comment:
Typo should be //MD5Base64String returns base64 encoded MD5sum value

Reply:
Fixed!